### DISCUSSION TITLE
Whether it can support `golang` as server rendering

### DISCUSSION THREAD

**Discussion by HomWang** [withastro/roadmap#509] [Proposal]
### Body

# Summary

A brief, one or two sentence explanation of the proposal.

# Background & Motivation

Include any useful background detail that that explains why this RFC is important.
What are the problems that this RFC sets out to solve? Why now? Be brief!

It can be useful to illustrate your RFC as a user problem in this section.
(ex: "Users have reported that it is difficult to do X in Astro today.")

# Goals

A **concise, bulleted-list** outlining the intended goals of this RFC. 

- What are the exact problems that you are trying to solve with this RFC?
- Separate these goals from the solution that you will outline below.
- If this RFC isn't approved, these goals can be used to develop alternative solutions.

# Example

If the proposal involves a new or changed API, then include a basic code example.
Otherwise, omit this section if it's not applicable.

**Comment by bluwy** [comment]
Astro's renderer is written in JavaScript, and the code you write is written in JavaScript too. I don't think it's possible to run it entirely in go unless you run a JS vm in Go, which kinda defeats the purpose.
